background.js

```diff
@@ -5,12 +5,18 @@ window.updatePopup = null;
 let webPageConnection = null;
 let socket = null;
 let roomId = null;
+let skipIntro = true;
```

Comment:
I think it might be better to have null as the initial state so we know that if skipIntro === null, it haven't being loaded yet

Reply:
Ok